messenger/http_transporter.go

```diff
@@ -44,6 +44,18 @@ var (
 	errNotStarted      = errors.New("HTTP transport has not been started")
 	errTerminal        = errors.New("HTTP transport is terminated")
 	errAlreadyRunning  = errors.New("HTTP transport is already running")
+
+	httpTransport, httpClient = &http.Transport{
```

Comment:
it might. i did not test.

On Fri, Oct 16, 2015 at 12:20 PM, Karl Isenberg <notifications@github.com>
wrote:

> In messenger/http_transporter.go
> <https://github.com/mesos/mesos-go/pull/198#discussion_r42260784>:
>
> > @@ -44,6 +44,18 @@ var (
> >  	errNotStarted      = errors.New("HTTP transport has not been started")
> >  	errTerminal        = errors.New("HTTP transport is terminated")
> >  	errAlreadyRunning  = errors.New("HTTP transport is already running")
> > +
> > +	httpTransport, httpClient = &http.Transport{
>
> Are you positive this doesn't work as two declarations? I could sworn I'd
> done that before for this exact reason.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesos/mesos-go/pull/198/files#r42260784>.
>